.github/workflows/zizmor.yml

```diff
@@ -0,0 +1,33 @@
+name: GitHub Actions Security Analysis with zizmor 🌈
+
+on:
+  push:
+    branches: ["master", "release-**", "staging-**"]
+  pull_request:
+    branches: ["master", "release-**", "staging-**"]
```

Comment:
Can this only run if we actually edit anything in .github/workflows?

Reply:
I'm not sure if that would work well with https://github.com/NixOS/nixpkgs/pull/396451#issuecomment-2781154695, given the docs imply that the scan needs to run for the PR to be merged if we enable it.